### DISCUSSION TITLE
我的图片-重命名失败，提示服务异常，请稍后重试。

### DISCUSSION THREAD

**Discussion by toreyink** [lsky-org/lsky-pro#557] [问答]
如图，上传正常，但无法重命名。
![image](https://user-images.githubusercontent.com/116132240/198916346-8ae0a23b-36a5-4419-9e78-5938b52a18b5.png)

**Comment by 0xxb** [comment]
已经修复过了，下个版本中发布。
duplicate of #536

**Comment by izelle** [comment]
我和你是一样的，另外企业版在哪里下载呢
